test.ps1

```diff
@@ -62,8 +62,8 @@ Get-Module Pester | Remove-Module
 
 if (-not $SkipPTests) {
     $result = @(Get-ChildItem $PSScriptRoot/tst/*.ts.ps1 -Recurse |
-            foreach {
-                $r = & $_.FullName -PassThru -NoBuild:$NoBuild
+            ForEach-Object {
+                $r = & $_.FullName -PassThru -NoBuild:$true
```

Comment:
Is this on purpose?

Reply:
Yeah, removed use of alias. PSSA-warning + confusing 🙂